### PR TITLE
Adding discard_after_delete parameter for storages

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

File: roles/oVirt.storages/tasks/main.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@
     posixfs: "{{ item.value.posixfs | default(omit) }}"
     glusterfs: "{{ item.value.glusterfs | default(omit) }}"
     fcp: "{{ item.value.fcp | default(omit) }}"
+    discard_after_delete: "{{ item.value.discard_after_delete | default(omit) }}"
```
Comment (machacekondra) [inline roles/oVirt.storages/tasks/main.yml:19]: should be default(true) ?

Comment (petr-balogh) [reply roles/oVirt.storages/tasks/main.yml:19]: It's by default true in module side, so I would omit it here if it's not specified in our config and let module to do the default.

Comment (petr-balogh) [reply roles/oVirt.storages/tasks/main.yml:19]: Changed to undef after offline discussion.

**Comment by petr-balogh** [comment]
Hey, can we merge this PR? It's verified and working well. Thx

**Review by machacekondra — APPROVED** [review]

**Pull request merged**